diff -Nurp docker-2.4.2.orig/setup.py docker-2.4.2/setup.py
--- docker-2.4.2.orig/setup.py	2017-06-27 19:56:40.000000000 -0500
+++ docker-2.4.2/setup.py	2017-07-25 13:21:36.725686849 -0500
@@ -5,17 +5,8 @@ import codecs
 import os
 import sys
 
-import pip
-
 from setuptools import setup, find_packages
 
-if 'docker-py' in [x.project_name for x in pip.get_installed_distributions()]:
-    print(
-        'ERROR: "docker-py" needs to be uninstalled before installing this'
-        ' package:\npip uninstall docker-py', file=sys.stderr
-    )
-    sys.exit(1)
-
 ROOT_DIR = os.path.dirname(__file__)
 SOURCE_DIR = os.path.join(ROOT_DIR)
 
